tests/test_dummy.py

```diff
@@ -63,12 +65,12 @@ def test_dummy_execute_pulse_sequence_unrolling(
     sequence = PulseSequence()
     sequence.concatenate(natives.single_qubit[0].MZ.create_sequence())
     for _ in range(nsequences):
-        sequences.append(sequence)
+        sequences.append(deepcopy(sequence))
```

Comment:
Isn't this equivalent to just
```py
for _ in range(nsequences):
    sequences.append(natives.single_qubit[0].MZ.create_sequence())
```
?

Reply:
Yes, it is. I've just been lazy, once more. Let me fix it.

Reply:
Done in a7ef4ec